<?xml version="1.0"?>

<launch>

    <!-- Turtlesim Node -->
    <node pkg="turtlesim" type="turtlesim_node" name="sim" />

    <node pkg="turtlesim" type="turtle_teleop_key" name="teleop" output="screen" />


    <!-- Axes -->
    <param name="scale_linear" value="0.5" type="double"/>
        
    <param name="scale_angular" value="0.5" type="double"/>

    <node name="turtle1_tf_broadcaster" pkg="learning_tf" type="turtle_tf_broadcaster" args="/turtle1"/>
   
    <node name="tuttle2_tf_broadcaster" pkg="learning_tf" type="turtle_tf_broadcaster"  args="/turtle2"/>
    
    <node name="listener" pkg="learning_tf" type="turtle_tf_listener" />

    <node name="broadcaster_frame" pkg="learning_tf" type="frame_tf_broadcaster"/>
    
    
    
</launch>